src/app/app.routes.ts

```diff
@@ -84,6 +85,10 @@ export const ROUTES: Routes = [
     path: 'terms-and-conditions',
     component: TermsAndConditionsComponent
   },
+  {
+    path: 'terms-and-conditions-register-other-org',
+    component: TermsAndConditionsRegisterOtherOrgComponent
```

Comment:
Minor comment: Really long component name and path. Could we shorten it?

Reply:
I started the development with TermsAndConditions**New**Component, but it did not sound like a suitable name so changed it to this lengthy name.

We will leave it to the future developers to rename when the Business wants to get rid of the current Terms and Conditions and use this new one instead :)